src/main/java/com/btkelly/gnag/reporters/BaseReporter.java

```diff
@@ -95,8 +99,22 @@ private T getReport(Project project) throws JAXBException {
         }
     }
 
-    private java.io.File getReportFile(Project project) {
-        return new java.io.File(project.getProjectDir(), getReportFilePath());
+    @Nullable
+    private java.io.File getReportFile(Project project) throws FileNotFoundException {
```

Comment:
Have you checked all the logic around this change still holds? This can now return a null, before it could not. And there are checks to see if the file exists which would expect the report file to always be non null